src/components/StakingPositionsList/StakingPositionsList.tsx

```diff
@@ -135,10 +136,33 @@ export const StakingPositionsList: React.FC = () => {
     navigate("/")
   }, [navigate])
 
+  const mapleAlertVisible = useMemo(() => positions?.some((item) => item.id < 25), [positions])
```

Comment:
It should be more than 25 right?

Reply:
Yes, it was just a temporary value until we find the exact ID of the last NFT that is impacted by Maple.

Reply:
We can find it in the database, maple strategy was removed last night, all those positions were affected